pkg/events/parse_args_helpers.go

```diff
@@ -213,7 +215,7 @@ func parseBpfHelpersUsage(arg *trace.Argument, helpersList []uint64) {
 	}
 
 	arg.Type = "const char**"
-	arg.Value = usedHelpers
+	arg.Value = fmt.Sprint(usedHelpers)
```

Comment:
The `usedHelpers` slice is already the finalized value, it shouldn't be formatted as a string. Instead, we should handle parsing errors inside the loop by adding the raw helper number to the slice (formatted as a string)

Reply:
@ShohamBit The issue isn't solved for this function yet because helpers with parsing errors will not be included in the list. We should add the raw value for helpers that couldn't be parsed.